flow/metrics.yaml

```diff
@@ -69,3 +81,44 @@ components:
               x-field-uid: 1
             cut_through:
               x-field-uid: 2
+    Flow.TxRxReplication:
```

Comment:
Wouldn't `Replication` imply additional copies? Meaning, if we have `Replication = 1` it means we expect to receive twice more packets that we sent? Maybe we could use `TxRxRatio` instead?

Reply:
should we change the reference as well from `$ref: '#/components/schemas/Flow.TxRxReplication'` to `$ref: '#/components/schemas/Flow.TxRxRatio'`?

Reply:
Updated.